store/imagestore/remote.go

```diff
@@ -19,9 +19,14 @@ package imagestore
 
 import (
 	"database/sql"
+	"errors"
 	"time"
 )
 
+var (
+	ErrRemoteNotFound = errors.New("remote with the given ACI URL not found")
```

Comment:
I don't agree with returning `nil` to check whether a function succeeded or failed. We already have an `error` field in the function signature